typescript/web/src/components/labeling-tool/openlayers-map/__tests__/draw-bounding-box-and-polygon-interaction.tsx

```diff
@@ -132,9 +132,14 @@ it("is possible to undo the creation of the label", async () => {
 
   await useUndoStore.getState().undo();
 
+  // @ts-ignore
```

Comment:
@RaynalHugo Why is there a `ts-ignore` here? Can't you use TS typings to fix the compiler issue?

Reply:
Typescript doesn't understand itself that we called jest.mock on the apollo client. 

I removed the ts-ignore, but replaced it with `as jest.Mock` on the apollo client. Apparently, that's the standard way to deal with mocks. 

It's not really type-safe. After checking what it does, everything after `calls[0][0]` is typed as any. 
I think `jest.Mock` is a generic and I could somehow type this. But I don't think this is necessary to go further on this. It's only used on tests and that is basically what we want to assert on the `expect` on the following lines.

Reply:
@bbenoist I am merging the PR now as it is blocking, but, if you want, we can continue discussing this here and implement the needed changes on another PR.